runtime/js/10_permissions.js

```diff
@@ -5,7 +5,7 @@
   const {
     Event,
     EventTarget,
-    Deno: { core },
+    Deno: { core: { ops } },
```

Comment:
```suggestion
    Deno: { core, core: { ops } },
```
This is causing the failures, they're `ReferenceError`s for `core`.